js/plugin/popups.ts

```diff
@@ -129,18 +129,15 @@ export interface BrowserModPopupParams {
 const customElementClassCache: Record<string, typeof BrowserModPopup> = {};
 
 export function setCustomElementClass(dialogTag: string): void {
-  if (customElementClassCache[dialogTag]) {
+  // Check if already registered in customElements registry (not just cache)
```

Comment:
All these changes for custom class are unneccessary. If they are to fix any issue (unlikely) then please submit as separate PR.